django/test/selenium.py

```diff
@@ -191,6 +191,26 @@ def dark(self):
             finally:
                 self.selenium.execute_script("localStorage.removeItem('theme');")
 
+    def set_emulated_media(self, features, media=""):
+        if self.browser != "chrome":
+            return
```

Comment:
I guess we could skip on unsupported browsers to avoid generating misleading screenshots? 🤔

```suggestion
            self.skipTest("Emulated media controls are only supported on Chrome.")
```

Reply:
That's much nicer thank you